operator/pkg/reconciliation/reconcile_racks.go

```diff
@@ -473,6 +487,9 @@ func (rc *ReconciliationContext) CheckRackScale() result.ReconcileResult {
 				"desiredSize", desiredNodeCount,
 			)
 
+			rc.Recorder.Eventf(rc.Datacenter, "ScaleUpRack",
```

Comment:
According to https://godoc.org/k8s.io/client-go/tools/record#EventRecorder , the "type" argument can only be "Normal" or "Warning".

Reply:
Err, yeah, that's a bug.